src/components/socio-carddesktop.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+const ColaboradorCardDesktop: React.FC = () => {
+  return (
+    <div className="relative flex h-[192px] w-full max-w-[1300px] items-center overflow-hidden rounded-xl bg-blue-600 p-6 text-white">
```

Comment:
We should use the same blue the Figma design uses and to make things more clean we should even add it as a custom color in `tailwind.config.js` under `colors` so we can use it easily if we need it. Changing the background to the same color also helps the image in the component fit better with the background.